Externals_CAM.cfg

```diff
@@ -58,15 +58,19 @@ required = True
 
 [camnor_phys]
 protocol = git
-tag = camnor_noresm_v0.0.5
-repo_url = https://github.com/NorESMhub/CAM-Nor-physics
+#tag = camnor_noresm_v0.0.5
+#repo_url = https://github.com/NorESMhub/CAM-Nor-physics
+branch = feature/update_noresm_develop_to_noresm2_3_develop
+repo_url = https://github.com/mvertens/CAM-Nor-physics
 local_path = src/physics/camnor_phys
 required = True
 
 [oslo_aero]
 protocol = git
-tag = noresm_oslo_aero_v0
-repo_url = https://github.com/NorESMhub/OSLO_AERO
+#tag = noresm_oslo_aero_v0
+#repo_url = https://github.com/NorESMhub/OSLO_AERO
+branch = feature/update_noresm_develop_to_noresm2_3_develop
```

Comment:
Is this branch still the correct branch?

Reply:
yes